components/User/UserCard.tsx

```diff
@@ -40,15 +41,18 @@ const UserCard: FC<Props> = ({ user }) => {
             position="absolute"
             bottom={0}
             transform="translate(1rem, 50%)"
-            bg="black"
             border="2px solid"
             borderColor="white"
             rounded="full"
+            w={16}
+            h={16}
           >
-            <Avatar
-              size="lg"
-              src={user.image || undefined}
-              title={user.name || user.address}
+            <Box
+              as={AccountImage}
+              address={user.address}
+              image={user.image}
+              size={60}
```

Comment:
The image is not correctly displayed here https://demo-bsc-testnet-kp3u49l1c-liteflow.vercel.app/explore/users

```suggestion
              size={60}
              objectFit="cover"
```